.buildkite/pipeline.yml

```diff
@@ -3,6 +3,13 @@ env:
   RUSTFLAGS: -D warnings
 
 steps:
+  - label: "protobuf backward compatibility"
+    command: |
+      buf breaking --against '.git#branch=master'
+    timeout: 30
+    agents:
+    - "rust=1.61.0"
```

Comment:
Do we need rust version specified here? Will we need to ever update it? Can we just do `distro=amazonlinux` like all the other places?

Reply:
@anshal-savla has created a separate stack of buildkite agent containing the buf binary, and marked with this particular tag. Once stabilized, he will replace the default stack with this one and then this tag should be removed.